frontend/components/employee/employee.js

```diff
@@ -0,0 +1,201 @@
+import {Grid} from "@mui/material";
+import * as React from "react";
+import {useState} from "react";
+import {CustomDatePicker, CustomAutoComplete, InputTextField, DropDownList} from '../form/formComponent';
+import Button from "@mui/material/Button";
+import DialogActions from "@mui/material/DialogActions";
+
+
+const EmployeeForm = (props) => {
+
+    const titleList = [
+        {id: 0, value: "General Practice"},
+        {id: 1, value: "Acupuncture"},
+        {id: 2, value: "Manager"}
+    ];
+    const genderList = [
+        {id: '0', value: 'male', title: 'Male'},
+        {id: '1', value: 'female', title: 'Female'},
+        {id: '2', value: 'na', title: 'N/A'},
+    ];
+    const {
+        handleClose,
+        mode,
+        initValues,
+        tabValue,
+        editEmployee,
+        addEmployee,
+        serviceList,
+        serviceEmployeeList
+    } = props;
+    const [employeeValue, setEmployeeValue] = useState(initValues);
+    const [errorMessage, setErrorMessage] = useState([]);
+
+
+    const addNewEmployee = () => {
+        if (validate()) {
+            let service_ids = [];
+            for (let service of employeeValue.services) {
+                let idArray = service.durations_prices.map(durationprice => durationprice.id)
+                console.log(idArray);
```

Comment:
Is this to be removed? idem for line 52.

Reply:
yes, it is to removed.